src/airport.js

```diff
@@ -1 +1,49 @@
+class Airport {
+    hangar;
+    #maxCapacity;
+    weather;
 
+    constructor(plane = [], capacity = 3, weather = 'Clear') {
+        this.hangar = plane;
+        this.#maxCapacity = capacity;
+        this.weather = weather;
+    }
+
+    getWeather() {
+        return this.weather;
+    }
+
+    changeWeather(weather) {
+        weather === 'Stormy' ? this.weather = 'Stormy' : weather === 'Clear' ? this.weather = 'Clear' : console.log(`Please enter valid weather conditions. Valid conditions are 'Clear' or 'Stormy'.`);
+    }
+
+    getPlaneNum() {
+        return this.hangar.length;
+    }
+
+    getMaxCapacity() {
+        return this.#maxCapacity;
+    }
+
+    changeMaxCapacity(number) {
```

Comment:
EBI - could have added more caveats to this to count for outlying inputs. Could have used ternary statement for shorter code